src/main/java/net/openhft/chronicle/queue/internal/domestic/backed/map/QueueBackedMaps.java

```diff
@@ -10,6 +10,7 @@
 /**
  * Used by services to handle multiple QueueBackedMaps
  */
+@Deprecated(/* to be removed in x.25 */)
```

Comment:
These are no longer used in the codebase, and were only used by legacy QBM which was deprecated before and has now been removed, so appropriate to remove in x.25